codelingo/.lingo

```diff
@@ -1,8 +1,10 @@
-lexicons:
-- codelingo/common as _
 tenets:
-- name: find-funcs
-  doc: Example tenet that finds all functions.
-  comment: This is a function, but you probably already knew that.
-  match: |
-    <func
+  - name: find-funcs
```

Comment:
Same here, remove it.

Reply:
damn doing multiple things at once blocked my common sense